src/scss/skin-modern/components/_subtitlesettingspanel.scss

```diff
@@ -0,0 +1,53 @@
+@import '../variables';
+@import '../mixins';
+
+.#{$prefix}-ui-subtitles-settings-panel {
```

Comment:
What is this file necessary for? Why is the styling not covered by `_settingspanel.scss`?

Basically, for every SCSS in the `components` folder, there must be a `Component`, which in this case is violated. You should generify the `SettingsPanel` so it covers also the subtitle settings.

Reply:
It's needed for the different positioning and the little arrow to its toggle button.

Reply:
Ok but why is that whole file necessary to just change the positioning? Shouldn't that be much simpler?

Reply:
Should be. We can consider using something like a wrapper over the `SettingsToggleButton` and the `SettingsPanel` to 'group' then together. Than the positioning and the little arrow should be easier.

But i would say to do this in a separate issue and then first extract the `Panel` and introduce it also for the `SettingsPanel`

Reply:
Even without a wrapper, I don't think it's necessary to replicate that much of the `_settingspanel.scss`. This file also should not contain styles for a `ListBox`.